pkgs/applications/misc/rtfm/patches/make.patch

```diff
@@ -0,0 +1,30 @@
+--- a/Makefile	2023-07-17 17:18:28.000000000 +0300
++++ b/Makefile	2023-07-19 12:13:44.627168135 +0300
+@@ -4,8 +4,7 @@
+ all: configure .WAIT rtfm docsets
+ 
+ configure:
+-	shards install
+-	./bin/gi-crystal
++	gi-crystal
+ 
+ rtfm:
+ 	shards build --release -s rtfm
+@@ -29,13 +28,15 @@
+ 	install -D -m644 data/io.github.hugopl.rtfm.gschema.xml $(DESTDIR)$(PREFIX)/share/glib-2.0/schemas/io.github.hugopl.rtfm.gschema.xml
+ 	# docsets
+ 	mkdir -p $(DESTDIR)$(PREFIX)/share/rtfm/docsets/
+-	cp -rv data/Crystal.docset $(DESTDIR)$(PREFIX)/share/rtfm/docsets/
+-	cp -rv data/Gtk4.docset $(DESTDIR)$(PREFIX)/share/rtfm/docsets/
++	cp -r data/Crystal.docset $(DESTDIR)$(PREFIX)/share/rtfm/docsets/
++	cp -r data/Gtk4.docset $(DESTDIR)$(PREFIX)/share/rtfm/docsets/
```

Comment:
I added this change upstream at https://github.com/hugopl/rtfm/commit/8833114005f37b0ded3b62a8d296f02a0b1cfeaa